classes/Employee.php

```diff
@@ -507,10 +507,9 @@ public function logout()
     public function favoriteModulesList()
     {
         return Db::getInstance()->executeS('
-			SELECT `module`
-			FROM `'._DB_PREFIX_.'module_preference`
-			WHERE `id_employee` = '.(int)$this->id.' AND `favorite` = 1 AND (`interest` = 1 OR `interest` IS NULL)'
-        );
+		SELECT `module`
```

Comment:
Why did you change the indentation?

Reply:
@PierreRambaud 

It's my sublime pluggin but we found 3 idententation in this class : 

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L304

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L381

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L507

Tell me which is the right one

Reply:
This one look really good, https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Employee.php#L304
It's PSR2 / Symfony style compliant :)